README.md

```diff
@@ -1,2 +1,154 @@
-# python
-Python SDK to interact with the Asgardeo management SDK
+# Asgardeo Python SDKs
+
+Official Python SDKs for [Asgardeo](https://wso2.com/asgardeo/) - WSO2's Identity and Access Management platform.
+
+## Packages
+
+This repository contains two complementary Python packages:
+
+### [asgardeo](./packages/asgardeo/)
+Core Python SDK for Asgardeo authentication flows.
+
+```bash
+pip install asgardeo
+```
+
+**Features:**
+- ✅ Native authentication flows (no browser redirects)
+- ✅ Token management (exchange, refresh)
+- ✅ Full async/await support
+- ✅ Automatic resource cleanup
+- ✅ Type hints and error handling
+
+### [asgardeo-ai](./packages/asgardeo-ai/)
+AI agent authentication and on-behalf-of (OBO) token flows.
+
+```bash  
+pip install asgardeo-ai
+```
+
+**Features:**
+- ✅ AI agent authentication
+- ✅ On-behalf-of (OBO) token flows
+- ✅ User authorization URL generation
+- ✅ Built on top of core asgardeo SDK
+
+## Quick Start
+
+### Core Authentication
+```python
+from asgardeo import AsgardeoConfig, AsgardeoNativeAuthClient
+
+config = AsgardeoConfig(
+    base_url="https://api.asgardeo.io/t/your-org",
+    client_id="your_client_id",
+    redirect_uri="your_redirect_uri",
+    client_secret="your_client_secret"
+)
+
+async with AsgardeoNativeAuthClient(config) as client:
+    # Authenticate user
+    await client.authenticate()
+    await client.authenticate(
+        authenticator_id="BasicAuthenticator",
+        params={"username": "user@example.com", "password": "password"}
+    )
```

Comment:
Is two authenticate calls sequentially intentional?

Reply:
yes first one is initial one (/authorize) then (/authn)